src/Common/MoistThermodynamics/states.jl

```diff
@@ -276,8 +298,10 @@ function LiquidIcePotTempSHumNonEquil(
     ρ::FT,
     q_pt::PhasePartition{FT},
     maxiter::Int = 5,
-    tol::FT = FT(1e-1),
+    temperature_tol::FT = FT(1e-3),
 ) where {FT <: Real}
+    _cp_d = FT(cp_d(param_set))
+    tol = ResidualTolerance(temperature_tol*_cp_d)
```

Comment:
Again, we shouldn't need any of this. If we do, it should be 
```suggestion
    tol = ResidualTolerance(potential_temperature_tol)
```